src/claims.rs

```diff
@@ -353,4 +363,13 @@ mod tests {
             ValidationError::NotMature
         );
     }
+    #[test]
+    fn float_timestamp() {
```

Comment:
You can also add a couple of overflowing timestamps (such as `{"exp": 1e64}` and `{"exp": -1e64}`) to test that they lead to a deserialization error.